libr/debug/debug.c

```diff
@@ -1561,7 +1561,17 @@ static int show_syscall(RDebug *dbg, const char *sysreg) {
 	return reg;
 }
 
-// continue execution until a syscall is found, then return its syscall number or -1 on error
+/* Continue execution until a syscall is found.
+ *
+ * If no plugin is selected, read the x86 sn register in order to determine the syscall. See https://www2.lauterbach.com/pdf/debugger_x86.pdf.
+ * In case of bug, it is recommended to use a debugger plugin such as -d "windbg://file.exe" on Windows.
+ *
+ * first argument, RDebug: debugger configuration structure.
+ * second argument, int sc: register value.
```

Comment:
first, second, third, those are not correct ways to refere to the arguments, varnames have names, and those should be clear anough to not need to be documented for example. its part of the developer guide

Reply:
good point. does r2 still uses doxygen?

Reply:
I need more documentation experience.

Reply:
R2 never used doxygen and never enforced documentation. See DEVELOPERS.md 

so no need to document anything. Is something is not obvious change function names and variables but documentation shouldnt be added

Reply:
ah my bad. I suggest to add reference doc... maybe for a full PR. What do you think of this idea?